### ISSUE TITLE
Command line to debug submission

### ISSUE THREAD

**Issue by stefano-maggiolo** [cms-dev/cms#1008]
**Is your feature request related to a problem? Please describe.**
Reevaluate a testcase or the full set of testcases keeping the sandbox to inspect output files and so on. Intended for handling appeals.

**Describe the solution you'd like**
From the brainstorm on gitter:

we would add a new command line tool, say cmsReRunSubmission
that takes a submission id (and, maybe, an optional testcase id and stuff like that)
runs the submission on a worker but doesn't store the result (so, in particular, it doesn't overwrite the score)
instead it prints the new results and the paths to the sandboxes where you can find the data
the sandbox would of course reside on a worker, but maybe we could even think of adding a --local option that runs the evaluation locally
(does it make sense? the conditions would be different but it may be easier to debug)

basically bypassing ES, no?

doesn't it at least have to use the workerpool and the queue of ES?

oh right
we need a way to retrieve the sandbox paths then

uhm right
we would anyway have to add a new rpc endpoint to es

maybe a new "opaque job injection" to ES...

exactly

**Comment by lerks** [comment]
Duplicate of #1007? :P

**Comment by stefano-maggiolo** [comment]
Ups.
